angularjs/Signature/How-To.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Syncfusion Signature How To
-description: How To
-platform: AngularJS
+title: How To Section in AngularJS Signature | Syncfusion
+description: Learn here all about how to section Syncfusion AngularJS Signature component, its elements and more.
+platform: angularjs
 control: Signature
 documentation: ug
 ---
 
-## How To?
+# How To Section in AngularJS Signature
 
-### Save signature image with user defined format
+## Save signature image with user defined format
 
 By default, the downloaded image from the signature canvas will be in **png** format. We can define our own format to download the image with [`e-saveimageformat`] property. And we can also save the image along with the background by using the **e-savewithbackground** property.
 
```

Comment:
Below H3 should be H2 only. so change it as H2 and share.

Reply:
changed